ci-operator/jobs/openshift/external-dns-operator/openshift-external-dns-operator-release-1.1-presubmits.yaml

```diff
@@ -0,0 +1,482 @@
+presubmits:
+  openshift/external-dns-operator:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^release-1\.1$
```

Comment:
Shouldn't the presubmit happen off the master branch?

Reply:
Why should it happen off the master branch for `release-1.1` branch?
Btw this file was generated by `make jobs` off the config I added, so I guess it's the normal setup for a new branch.

Reply:
I think the release-*.* code is generated using the *-master-presubmits.yaml, or maybe [openshift-external-dns-operator-main.yaml](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/external-dns-operator/openshift-external-dns-operator-main.yaml) in this case.

Reply:
Something is off - take a look at the structure in https://github.com/openshift/release/tree/master/ci-operator/config/openshift/external-dns-operator

Reply:
Sorry, I'm not getting the point. To be the presubmits which target branch `release-1.1` should happen off `release-1.1` branch. The release before the previous release (`1.0` is missing the CI config btw) was dong [the same](https://github.com/openshift/release/blob/615429eab1c4e444f0ac9ac32289deeba136008b/ci-operator/jobs/openshift/external-dns-operator/openshift-external-dns-operator-release-0.1-presubmits.yaml#L6-L7). The cluster-ingress-operator's 4.11 release branch presubmit [uses](https://github.com/openshift/release/blob/615429eab1c4e444f0ac9ac32289deeba136008b/ci-operator/jobs/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-release-4.11-presubmits.yaml#L6-L7) `release-4.11` branches too.

Reply:
Was this file generated or created manually?

Reply:
Generated, by `make jobs` from the config I put in this PR. I believe it's just the naming convention of the config file which contains the branch name.

Reply:
This looks all good, but there should be another config testing the `main` branch (there's no `master` in https://github.com/openshift/external-dns-operator/)